libr/bin/format/elf/elf.h

```diff
@@ -39,6 +39,7 @@ typedef struct r_bin_elf_symbol_t {
 	int last;
 	bool in_shdr;
 	bool is_sht_null;
+	bool is_value;  /* when true, sym has no paddr, only vaddr */
```

Comment:
i still think this is absolutely not a good name. We can work on this in separate PR, but given the comment, I think `has_vaddr_only` is a better name.

Reply:
ok for separate PR